contrib/IECoreAlembic/include/IECoreAlembic/AlembicInput.h

```diff
@@ -49,12 +49,7 @@ namespace IECoreAlembic
 
 IE_CORE_FORWARDDECLARE( AlembicInput )
 
-/// This class provides very high level access to the contents
-/// of an Alembic cache. It deliberately hides all Alembic
-/// data types and provides an interface tailored to the
-/// reading of 3D scene data only. Finer control and lower level
-/// access can be obtained by using a combination of the Alembic
-/// APIs and the FromAlembicConverters.
+/// \todo Remove
```

Comment:
I'd like to do at least one release of Gaffer where SceneReader has .abc support and the AlembicSource node still exists, so any interested folks can compare performance and also get to grips with the new approach where a single location can have a transform and an object.

The other thing I need to do is to promote IECoreAlembic out of contrib - I was anticipating doing them both in the same PR.